circuitbreaker.py

```diff
@@ -15,28 +15,81 @@
 except ImportError:
     from monotonic import monotonic
 
+# Python2 vs Python3 strings
+try:
+    basestring
+    STRING_TYPES = (basestring, unicode)
+except NameError:
+    STRING_TYPES = (bytes, str)
+
+
 STATE_CLOSED = 'closed'
 STATE_OPEN = 'open'
 STATE_HALF_OPEN = 'half_open'
 
+def in_exception_list(*exc_types):
+    """Build a predicate function that checks if an exception is a subtype from a list"""
+    def matches_types(thrown_type, thrown_value):
```

Comment:
`thrown_value` is not used, can we drop it?

Reply:
I have changed the second arg to `_` to avoid any linter warnings. The function signature has to have two arguments to support the case where someone wants to use the thrown value.